recipes/ninja_syntax/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "ninja_syntax" %}
+{% set version = "1.6.0" %}
+{% set sha256 = "33c203b4ee39a638c9af757ed1c4257e05abf1fae0615305d103020f36c804c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/43/d2/7a9cafe2b3b6ee260dfeee1a963630c8f0c16620c0ab7d18caab7fb46a42/{{ name }}-{{ version }}.tar.gz
+
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - ninja_syntax
+
+about:
+  home: https://pypi.python.org/pypi/ninja_syntax/1.6.0
+  license: Apache-2.0
```

Comment:
Ok, license file is not in the source archive, but in the repo.

I'll run into the same issue as [git-lfs recipe](https://github.com/conda-forge/staged-recipes/pull/2015).